azure-pipelines/templates/stage-artifacts.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  - task: CopyFiles@2
+    displayName: "\U0001F449 Copy packages and vsix to staging directory"
+    inputs:
+      # Uploading the package.json so we can publish later without cloning the source
```

Comment:
Can you elaborate more on this?

Reply:
In the release pipeline I'm testing, I'm not cloning the source at all. I'm just downloading and releasing artifacts from a build pipeline run. In the release pipeline I need the package.json to know what extension is being released.